packages/web-runtime/src/pages/account.vue

```diff
@@ -159,18 +156,18 @@ export default defineComponent({
     const disableEmailNotificationsValue = ref<boolean>()
 
     // FIXME: Use graph capability when we have it
-    const isSettingsServiceSupported = useCapabilitySpacesEnabled()
-    const isChangePasswordEnabled = useCapabilitySpacesEnabled()
+    const spacesEnabled = useCapabilitySpacesEnabled()
```

Comment:
I liked the aliases of the spaces enabled capability. Reason: It's bad that we use spaces capability for the respective checks. But the aliases limited the defect to a scope where it's easy to understand with the comment above. Using the spaces capability in the template makes it harder to understand that that's not desired...

Reply:
Oh well, okay 😅  Then I'll add it a third time.